docs/neofs-blockstorage.md

```diff
@@ -108,11 +107,17 @@ OPTIONS:
 
 This command works as follows:
 1. Fetches the current block height from the RPC node.
-2. Searches for the oldest half-filled batch of block objects stored in NeoFS. 
-3. Fetches missing blocks from the RPC node and uploads them to the NeoFS container 
-starting from the oldest half-filled batch.
-4. After uploading the blocks, it creates index files for the newly uploaded blocks. 
-5. Uploads the created index files to the NeoFS container.
+2. Searches for the index files stored in NeoFS.
+3. Searches for the stored blocks for the next index file. 
+4. Fetches missing blocks from the RPC node and uploads them to the NeoFS container.
+5. After uploading the blocks, it creates index file from the found (step 3) 
+ and newly uploaded blocks. 
```

Comment:
> from the found

based on the found

> block OIDs.

Reply:
uploaded?

Reply:
"it creates index file based on the found (step 3) and newly uploaded block OIDs. "

Something like this.